.env.test

```diff
@@ -0,0 +1,29 @@
+# https://create-react-app.dev/docs/advanced-configuration
+GENERATE_SOURCEMAP=true
+
+# These API keys are intentionally public. Please do not report them - thank you for your concern.
+# REACT_APP_AMPLITUDE_PROXY_URL="https://api.uniswap.org/v1/amplitude-proxy"
+REACT_APP_AWS_API_REGION="us-east-2"
+REACT_APP_AWS_API_ENDPOINT="https://beta.api.uniswap.org/v1/graphql"
+REACT_APP_TEMP_API_URL="https://temp.api.uniswap.org/v1"
+REACT_APP_SENTRY_DSN="https://a3c62e400b8748b5a8d007150e2f38b7@o1037921.ingest.sentry.io/4504255148851200"
+REACT_APP_SENTRY_ENABLED=false
+ESLINT_NO_DEV_ERRORS=true
+REACT_APP_INFURA_KEY="f3a3e48678784c248ff32554efe73143"
+REACT_APP_MOONPAY_API="https://api.moonpay.com"
+REACT_APP_MOONPAY_LINK="https://us-central1-uniswap-mobile.cloudfunctions.net/signMoonpayLinkStaging?platform=web"
+REACT_APP_MOONPAY_PUBLISHABLE_KEY="pk_test_DycfESRid31UaSxhI5yWKe1r5E5kKSz"
+
+REACT_APP_VIOLET_EAT_SIGNER_TEST_PRIVATE_KEY=""
+
+# VIOLET WORKSPACE
+#
+## BUILD TIME VARS
+NPM_TOKEN="CHANGEME"
+REACT_APP_VIOLET_ENV="local"
+# this client id works for the local violet-app setup by default.
+# needs to be changed for the docker setup
+REACT_APP_VIOLET_CLIENT_ID="1212121212121212121212121212121212121212121212121212121212121214"
```

Comment:
Just checking if this is the intended default value?
Whenever we run tests in Github actions, don't we default to docker? If so, the default should probably be docker.